gamehub-backend/src/main/java/com/example/gamehubbackend/GamehubBackendApplication.java

```diff
@@ -12,4 +15,10 @@ public static void main(String[] args) {
 		SpringApplication.run(GamehubBackendApplication.class, args);
 	}
 
+	@Bean
+	public PasswordEncoder passwordEncoder() {
```

Comment:
just curious, is there a benefit/reason to not have the passwordEncoder in SecurityConfig?